taichi/transforms/insert_scratch_pad.cpp

```diff
@@ -73,8 +73,9 @@ class AccessAnalysis : public IRVisitor {
       offsets.resize(ptr->indices.size());
       int num_indices = (int)ptr->indices.size();
       for (int i = 0; i < num_indices; i++) {
-        auto diff = irpass::analysis::value_diff(ptr->indices[i], l,
-                                                 for_stmt->loop_vars[i]);
+        // TODO: StructForStmt::loop_vars is deprecated
+        auto diff = irpass::analysis::value_diff(ptr->indices[i], l, nullptr);
+        //                                       for_stmt->loop_vars[i]);
```

Comment:
remove?

Reply:
`insert_scratch_pad.cpp` is unused as we don't have vectorization now. But we want to add it back in the future, and I don't know how to modify this part.

Reply:
Sorry, I mean to remove the specific line L78...

Reply:
I see. I think it's easier to know what the code does if the original code is not removed.